wordpress-mobile/MediaPicker-iOS
# Update pod to version 1.0

This PR updates the pod to version 1.0

To test:
 - Check Podfile
 - Check Readme
 - Check Changelog
 - Run sample app just to see if all is working.
